snuba/subscriptions/scheduler.py

```diff
@@ -323,6 +347,7 @@ def __get_subscriptions(self) -> List[Subscription]:
         return self.__subscriptions
 
     def find(self, tick: Tick) -> Iterator[ScheduledSubscriptionTask]:
+        self.__reset_builder()
```

Comment:
This is executed once per message on the commit log, right ?

Reply:
Yes, that's how the ticks get created. One per per message.